### PR TITLE
fix: Work with Crystal v0.34.0

### PR DESCRIPTION
**Pull request by max-bertinetti**
Hi @jwoertink. 

I'm start a work to update fez. With this little modification it seems to work with the latest Crystal.

My next goal is to extend fez to work also with @anykeyh Clear ORM and @gdotdesign Mint language for a full stack solution for the web.

### CODE REVIEW

File: shard.yml [hunk 1]
```diff
@@ -11,4 +11,3 @@ targets:
 
 crystal: 0.22.0
```
Comment (jwoertink) [inline shard.yml]: Can you update this to be 0.34.0?

**Review by jwoertink — CHANGES_REQUESTED** [review]
Cool. I think just 1 small change, and this will be good to go.

File: shard.yml (hunk 1 above)
Comment (max-bertinetti) [reply shard.yml]: Ops! Done!

**Review by jwoertink — APPROVED** [review]
Thanks for doing this!

**Pull request merged**